controllers/operatorpolicy_controller.go

```diff
@@ -5,20 +5,46 @@ package controllers
 
 import (
 	"context"
+	"fmt"
+	"time"
 
+	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
+	policyv1 "open-cluster-management.io/config-policy-controller/api/v1"
 	policyv1beta1 "open-cluster-management.io/config-policy-controller/api/v1beta1"
 )
 
+const (
+	OperatorControllerName string = "configuration-policy-controller"
+	OperatorCRDName        string = "configurationpolicies.policy.open-cluster-management.io"
```

Comment:
Nit: `OperatorCRDName` isn't used, and `OperatorControllerName` should probably be "operator-policy-controller"

Reply:
Whoops, that's slightly embarrassing